src/library/scala/collection/mutable/UnrolledBuffer.scala

```diff
@@ -423,3 +425,10 @@ object UnrolledBuffer extends StrictOptimizedClassTagSeqFactory[UnrolledBuffer]
       array.take(size).mkString("Unrolled@%08x".format(System.identityHashCode(this)) + "[" + size + "/" + array.length + "](", ", ", ")") + " -> " + (if (next ne null) next.toString else "")
   }
 }
+
+// This is used by scala.collection.parallel.mutable.UnrolledParArrayCombiner:
+// Todo -- revisit whether inheritance is the best way to achieve this functionality
+private[collection] class DoublingUnrolledBuffer[T](implicit t: ClassTag[T]) extends UnrolledBuffer[T]()(t) {
```

Comment:
what's the reason for this collection being defined here rather than in the parallel collections?

Reply:
Good point, there is no reason, I should be able to move it to scala-parallel-collections.

Reply:
Actually, keeping `DoublingUnrolledBuffer` here allows `UnrolledBuffer` to be sealed. But anyway, we should probably move everything to scala-parallel-collections, right?

Reply:
I think it's ok to keep it here due to the tight integration, and tracking the usage in the comment as you did.